docs-v2/content/en/docs/builders/build-environments/cloud-build.md

```diff
@@ -39,16 +43,17 @@ The following options can optionally be configured:
 
 ## Faster builds
 
-Skaffold can build multiple artifacts in parallel, by settings a value higher than `1` to `concurrency`.
-For Google Cloud Build, the default is to build all the artifacts in parallel. If you hit a quota restriction,
-you might want to reduce  the `concurrency`.
+By default, Cloud Build builds all artifacts in parallel. Set `concurrency` to a non-zero value to specify
+the maximum number of artifacts to build concurrently. Consider reducing `concurrency` if you hit a quota
+restriction.
 
 {{<alert title="Note">}}
-When artifacts are built in parallel, the build logs are still printed in sequence to make them easier to read.
+When Skaffold builds artifacts in parallel, it still prints the build logs in sequence to make them easier to read.
```

Comment:
@aaron-prindle Would it be more correct to say "Cloud Build builds artifacts in parallel..." here?

Reply:
I think it makes sense to add somewhere that this is Cloud Build doing the builds in this sentence as you metioned, perhaps the wording can be:
 `Cloud Build (invoked by Skaffold) builds artifacts in parellel...`

Reply:
Done, thanks.